internal/api/v1/stage.go

```diff
@@ -41,8 +46,8 @@ func (app *stageParam) GitURL(server string) string {
 // ImageURL returns the URL of the image, using the ImageID. The ImageURL is
 // later used in app.yml.  Since the final commit is not known when the app.yml
```

Comment:
Since we are at it, this comment about the final commit seems well out of date (from when we were adding a commit on top).

Reply:
Done.